style.css

```
@@ -0,0 +1,1848 @@
+
+@media (min-width: 901px) {
+
+    .main-window {
+        width: min(58vw, 780px);
+        height: clamp(210px, 31vh, 315px);
+
+        /* Prevent the window from becoming too wide on
+           unusual browser dimensions */
+        max-width: 780px;
+
+        left: 50%;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)